src/cljam/io/vcf/writer.clj

```diff
@@ -49,64 +49,69 @@
     (cstr/upper-case (name k))
     (->camelCaseString k)))
 
+(defn- add-extra-fields
+  [fields m required-keys]
+  (reduce-kv (fn [fields k v]
+               (conj fields (str (->camelCaseString k) "=\"" v "\"")))
```

Comment:
Isn't `->PascalCaseString` more suitable? I couldn't find a mention of case for keys of extra fields from the spec, but many VCF seem to use `PascalCaseKey=value` style, except for contig extra fields. I feel it is more conventional to use `PascalCaseKey` by default and `camelCaseKey` for contig fields.

Reply:
I agree. I'll update the code so that the keys for extra fields will be emitted in PascalCase. Thanks!